### PR TITLE
New Decor

### PR DESCRIPTION
**Pull request by JustArt1m**
## About the PR
Several new decorative elements: floor lamps with random head color and blinds. Also a new hat: torchere. Floor lamps and blinds have crafts.

## Why / Balance
To make the station more cozy.

## Media
NewDecor

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Just_Art
- add: Added floor lamps!
- add: Added wooden and metal blinds!
- add: Added crafts for floor lamps and blinds!
- add: Added torchere hat!

### CODE REVIEW

**Comment by Hmeister-real** [comment]
i wish the lamp heads color changed the lights color

**Comment by JustArt1m** [comment]
> Not to nitpick too much, but I feel like the venetian blinds should be plastic, no?
> 
> Also, do they close?
> 
> Lamps look great btw.

The blinds are closing. Plastic ones would look almost the same as metal ones.

**Review by Tayrtahn — APPROVED** [review]

**Review by Tayrtahn — COMMENTED** [review]
Deconstruction graph needs fixing (not sure how that works).

Also, "torchere" is a pretty obscure word (if it even is one in English?).

**Comment by JustArt1m** [comment]
> Deconstruction graph needs fixing (not sure how that works).
> 
> Also, "torchere" is a pretty obscure word (if it even is one in English?).

Maybe I made a mistake, do you understand the word torchiere? If not, I'll replace it...

I checked, the deconstruction graph is working properly, what is the error?

**Comment by UbaserB** [comment]
> > Deconstruction graph needs fixing (not sure how that works).
> > Also, "torchere" is a pretty obscure word (if it even is one in English?).
> 
> Maybe I made a mistake, do you understand the word torchiere? If not, I'll replace it...
> 
> I checked, the deconstruction graph is working properly, what is the error?

replace torchiere i dont think many people know what the word means

**Comment by brainfood1183** [comment]
The worn lamp shade has the wrong directional sprite, looks like a goon style sprite which is more over head as opposed to perfectly straight on like the current ss14 sprites.

**Comment by IamVelcroboy** [comment]
Will you include a random spawner for the lamps too, please?

**Comment by JustArt1m** [comment]
> Will you include a random spawner for the lamps too, please?

The lamp color is random by default, no random spawner is required

**Comment by IamVelcroboy** [comment]
> > Will you include a random spawner for the lamps too, please?
> 
> The floor lamp color is random by default, no random spawner is required

Ooo fancy 👍

File: Resources/Prototypes/Entities/Structures/Doors/blinds.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  components:
+  - type: Sprite
+    sprite: Structures/Doors/Blinds/wood_blinds.rsi
+    state: frame
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: Construction
+    graph: Blinds
+    node: frameWood
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Structures/Doors/blinds.yml:81]: I believe you'll need to set deconstructionTarget on each of these to the appropriate base construction node.

File: Resources/Prototypes/Entities/Structures/Doors/blinds.yml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  id: BlindsWood
+  name: wooden blinds
+  parent: BaseBlinds
+  components:
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: Construction
+    graph: Blinds
+    node: startWoodBlinds
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Structures/Doors/blinds.yml:93]: Here too

File: Resources/Prototypes/Entities/Structures/Doors/blinds.yml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  components:
+  - type: Sprite
+    sprite: Structures/Doors/Blinds/metal_blinds.rsi
+    state: frame
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: FlimsyMetallic
+  - type: Construction
+    graph: Blinds
+    node: frameMetal
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Structures/Doors/blinds.yml:116]: Here too

File: Resources/Prototypes/Entities/Structures/Doors/blinds.yml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    drawdepth: BlastDoors
+    layers:
+    - state: closed
+      map: ["enum.DoorVisualLayers.Base"]
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: FlimsyMetallic
+  - type: Construction
+    graph: Blinds
+    node: startMetalBlinds
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Structures/Doors/blinds.yml:134]: Here too

**Review by Tayrtahn — CHANGES_REQUESTED** [review]

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
the worn lampshade is at the wrong perspective. I also think the blinds could be redone to more closely remember actual blinds. they look a lot like stairs.

**Comment by mirrorcult** [comment]
are you coming back to this?

**Comment by Emisse** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Pull request closed without merging**